getopt_compat.h

```diff
@@ -68,6 +68,8 @@
 #include <stdarg.h>
 #include <stdio.h>
 #include <windows.h>
+/* Required for JS_PRINTF_FORMAT */
```

Comment:
Can we not do this and silence the warnings if need be? I actually want to get rid of this header altogether by parsing command line arguments by hand in qjsc, since it's only used there...